Pellicola/Classes/PellicolaCache.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PellicolaCache.swift
+//  Pellicola
+//
+//  Created by marco.rossi on 17/10/2019.
+//
+
+import Foundation
+
+public final class PellicolaCache: NSObject {
+    private let fileHandler = PellicolaFileHandler()
+    
+    @objc public func clear() {
+        DispatchQueue.global(qos: .utility).async {
+            self.fileHandler.deleteAllImages()
```

Comment:
weak

Reply:
I'm not using weak reference to self here, cache object it's deallocated, after fileHandler deletions.